va/wayland/Makefile.am

```diff
@@ -65,5 +65,8 @@ EXTRA_DIST = \
 	wayland-drm.xml         \
 	$(NULL)
 
+BUILT_SOURCES = $(protocol_source_h)
+CLEANFILES = $(BUILT_SOURCES)
+
```

Comment:
Please check wayland-scanner in configure.ac for wayland platform. It is possible user doesn't install wayland-scanner.